src/Init/Core.lean

```diff
@@ -34,6 +36,25 @@ and `flip (·<·)` is the greater-than relation.
 
 attribute [simp] namedPattern
 
+/--
+`Empty.elim : Empty → C` says that a value of any type can be constructed from
+`Empty`.
```

Comment:
I think this is a useful addition for new users:

```suggestion
`Empty`. This can be thought of as a compiler-checked assertion that a code path is unreachable.
```